core/committer/txvalidator/v14/validator_test.go

```diff
@@ -1631,7 +1630,7 @@ func (exec *mockQueryExecutor) GetPrivateDataMetadata(namespace, collection, key
 }
 
 func createCustomSupportAndLedger(t *testing.T) (*mocktxvalidator.Support, ledger.PeerLedger, func()) {
-	ledgerMgr, cleanup := constructLedgerMgrWithTestDefaults(t, "txvalidator")
```

Comment:
sorry, possibly misguided Q: why remove the second argument?

Reply:
See the change at line 1951 below. `constructLedgerMgrWithTestDefaults` now uses the name from `t.TempDir()` as `testDir`.

```diff
- func constructLedgerMgrWithTestDefaults(t *testing.T, testDir string) (*ledgermgmt.LedgerMgr, func()) {
+ func constructLedgerMgrWithTestDefaults(t *testing.T) (*ledgermgmt.LedgerMgr, func()) {
```